File: SustitucionMOAModel/Dto/OrdenDeCarga/CrearOrdenDeCargaRequest.cs
using System.Collections.Generic;
using SustitucionMOAModel.Dto.OrdenDeCargaCommon;

namespace SustitucionMOAModel.Dto.OrdenDeCarga
{
    public class CrearOrdenDeCargaRequest
    {
        public Entities.OrdenDeCarga OrdenDeCarga { get; set; }

        public GestionAltasFAS GestionAltasFAS { get; set; }

        public List<UnidadTransporteCarga> UnidadesTransporte { get; set; }
    }
}

File: SustitucionMOA/src/app/common/models/ordenes-de-carga/crearOrdenDeCargaRequest.ts
import { UnidadTransporteCarga } from "../ordenes-de-carga-common/unidadTransporteCarga";
import { GestionAltasFAS } from "./gestionAltasFAS";
import { OrdenDeCarga } from "./ordenDeCarga";

export class CrearOrdenDeCargaRequest {

    public OrdenDeCarga: OrdenDeCarga;

    public GestionAltasFAS: GestionAltasFAS;

    public UnidadesTransporte: UnidadTransporteCarga[] = [];
}